### PR TITLE
WP-112 Implement dark mode

### PR DESCRIPTION
**Pull request by ericgmoon**
image
image
image
image
image

### CODE REVIEW

**Comment by ericgmoon** [comment]
Update: restyled drawer nav
image

**Review by s-hane — CHANGES_REQUESTED** [review]
Should be more dynamically applied rather than manual for each component

File: src/components/PageLayout/AppBar/index.styles.ts [hunk 1]
```diff
@@ -23,7 +25,7 @@ const Title = styled(Typography)`
 
 const StyledAppBar = styled(AppBar)`
   && {
-    background-color: ${({ theme }) => theme.palette.darkPrimary.main};
+    background-color: ${({ theme }) => (isDark(theme) ? theme.palette.backdrop.main : theme.palette.darkPrimary.main)};
```
Comment (ericgmoon) [inline src/components/PageLayout/AppBar/index.styles.ts:28]: This is the only place that requires manual adjustment. I couldn't find a nice way to make this part dynamic without compromising style. I think this should be the only place where this is necessary - the `AppBar` is such a large piece of the screen estate that I didn't want to compromise on design

**Comment by ericgmoon** [comment]
image
image

**Review by s-hane — APPROVED** [review]

**Pull request merged**